specification.md

```diff
@@ -202,39 +202,38 @@ Event data are stored in CSV files. Simple event data should be
 stored in a single column CSV, with each element in the array indicating the time of the
 event **relative to the start of the dataset**. The first line of the file must contain `start,`,
 indicating that the column contains the times of the event data. Event datasets can be
-distinguished from sampled datasets because the file is a plaintext CSV and `units` attribute must be
```

Comment:
This is a hold-over from Arf, where this was the only way to distinguish between sampled and event data. I feel like we should keep this in to ensure back compatibility. Also, I added the filetype attribute, which I'm not yet using (it's an idea at this point) but if you like that approach we could discuss that here.